src/components/Editor/Runners/HtmlRunner/HtmlRunner.jsx

```diff
@@ -61,7 +61,7 @@ function HtmlRunner() {
       (component) => `${component.name}.${component.extension}` === fileName,
     )[0];
 
-  const previewable = (file) => file.endsWith(".html");
+  const previewable = (file = "") => file.endsWith(".html");
```

Comment:
Would it be possible to add a test for this, @loiswells97? So that we don't accidentally break this again in future.

And out of interest, do you understand why the error is happening?

Reply:
It's a weird one, it only happens when the remix first loads up, but then if you reload the page it's fine... I suspect it might be some kind of weird race condition going on, something to do with the "open files" not having been opened before it tries to check whether the opened file can be previewed. If so, it's quite a nebulous thing to test to be honest with you - I was more just trying to build some more flexibility into the system so things can load properly

Reply:
agreed it would be nice to have a test around this, but can see it's a tricky one to write - we're a little up against it getting to this release and I can't see this making anything worse so I'll go ahead and accept if you're happy with that @chrisroos ?

Reply:
I've pushed an update to this one that fixes/works around the real issue around open files, so feel free to give it another review